addons/mail/static/src/models/chatter.js

```diff
@@ -33,17 +33,17 @@ registerModel({
         onAttachmentsLoadingTimeout() {
             this.update({ isShowingAttachmentsLoading: true });
         },
+        /**
+         * Handles click on the attach button.
+         */
+        onClickButtonAttach() {
```

Comment:
these two methods make the code confusing.... `onClickButtonAttach` and `onClickButtonAttachments`, if you really want to split them maybe it's a better idea to rename them into something more meaningful.

Reply:
I agree, the buttons have new names, so handler should also be adapted.